apps/mobile/src/queries/activity/account-activity.query.ts

```diff
@@ -1,9 +1,39 @@
+import { useAccountAddresses, useTotalAccountAddresses } from '@/hooks/use-account-addresses';
+import { toFetchState } from '@/shared/fetch-state';
 import { useSettings } from '@/store/settings/settings';
 import { QueryFunctionContext, useQuery } from '@tanstack/react-query';
 
-import { AccountAddresses } from '@leather.io/models';
+import { AccountAddresses, CryptoAssetInfo } from '@leather.io/models';
 import { getActivityService } from '@leather.io/services';
 
+/**
```

Comment:
I can fix this but this is actually a mistake I made when integrating activity. I should have used these hooks - `useTotalActivity` / `useAccountActivity` rather than the queries directly. 

I will fix it when adding the loading states in [LEA-1726](https://linear.app/leather-io/issue/LEA-1726/handle-loading-states-elegantly)